src/combobox/nls/ComboBox.ts

```diff
@@ -0,0 +1,9 @@
+const bundlePath = 'src/combobox/nls/common';
+const locales: string[] = [];
+
+const messages = {
+	clear: 'clear combo box',
```

Comment:
Same question I asked Neil, do we really need messages of this granularity, we could replace all the `open` / `clear` / `toggle` etc messages with a single `commonBundle`. Seems that we have a lot of repetition otherwise. If people want a custom message then I believe they can pass it as a widget property no?

Reply:
I tend to agree that messages should be injected into any component where the developer cannot directly modify the messages bundle. It would still be possible to set custom locale-specific messages, but doing so requires using `@dojo/i18n/i18n.setLocaleMessages` which isn't ideal, or users will try modifying the `locales` property of the bundle itself, which we also don't want.

Reply:
@tomdye I actually think it needs more than `open`/`clear`/`toggle`. If you're tabbing through without visual guidance, it's not always clear what controls are related to what widget. For the dialog, I played with the idea of something like `open ${title} dialog`, or even maybe `open ${title}`. I don't think that would get us to a single common bundle because ComboBox doesn't have a user-created title, and dialog's title is optional.

I'm open to thinking more about the organization though, if you think it's warranted. We could at least perhaps consolidate the words "open" and "close"